### PR TITLE
Check if value is None before validation

### LINKED ISSUE #1327: bngcentrepoint datatype can't be saved as None
Saving an empty National Grid References card through the UI (or through the ArcGIS Pro plugin 'create resource' feature) will produce the following error:

image

```
2024-09-05 13:49:18,733 arches.app.views.tile ERROR    Unable to save. Please verify your input is valid [Tile id: ]                      [Exception message: TileValidationError: Issue with input data]                      [Exception trace: Traceback (most recent call last):
  File "/home/samuel/arches_dev/arches_her/arches/arches/app/views/tile.py", line 144, in post
    tile.save(request=request, transaction_id=transaction_id)
  File "/home/samuel/arches_dev/arches_her/arches/arches/app/models/tile.py", line 459, in save
    self.validate([], request=request)
  File "/home/samuel/arches_dev/arches_her/arches/arches/app/models/tile.py", line 353, in validate
    raise TileValidationError(_("{0}".format(error_instance["message"])))
arches.app.models.tile.TileValidationError: 'Issue with input data'
]
Internal Server Error: /tile
```

### PR DESCRIPTION
**Pull request by samuelscandrett**
Closes #1327 

Checks if value is None before doing validation.  Allows empty BNG tiles to be saved.

### CODE REVIEW

File: arches_her/datatypes/bngcentrepoint.py [hunk 1]
```diff
@@ -96,16 +96,17 @@ def validate(self, value, row_number=None, source=None, node=None, nodeid=None,
             "TV",
             "TW",
         ]
-        try:
-            # CS - Validation for datatype.  Replicates functionality in widget which will be removed once datatype validation is fixed.
-            firstTwoCharacters = value[0:2]
-            numberElement = value[2:]
+        if value is not None:
```
Comment (csmith-he) [inline arches_her/datatypes/bngcentrepoint.py:99]: @SDScandrettKint Could you put a comment in the code to say this is a temporary fix until a change is made in core Arches?

**Review by csmith-he — COMMENTED** [review]
See request for comment to be made

**Comment by aj-he** [comment]
@SDScandrettKint - please also update the same file int he package... `arches_her/pkg/extensions/datatypes/bngcenterpoint/bngcentrepoint.py`

**Comment by samuelscandrett** [comment]
> @SDScandrettKint - please also update the same file int he package... `arches_her/pkg/extensions/datatypes/bngcenterpoint/bngcentrepoint.py`

Apologies, forgot about the package version.  Added in [3bfa6a3](https://github.com/archesproject/arches-her/pull/1328/commits/3bfa6a3e63eff32a8a2c81e5f48476348be41eec).

**Review by csmith-he — APPROVED** [review]

**Pull request merged**